docs/Project-Configuration/Project-Commands.md

```diff
@@ -1288,7 +1288,7 @@ and contains these fields:
 
 ## s3.get
 
-`s3.get` downloads a file from Amazon s3.
+`s3.get` downloads a file from Amazon s3. The download location is relative to [task's working directory](./Best-Practices.md#task-directory)
```

Comment:
Should we put this on the `local_file` parameter itself? This is just meant to be a summary of what the command does, and the parameters usually delve into the details of what's configurable by the user.

Same with the one below.